bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -181,6 +199,16 @@ public Builder setUseBulkApi(boolean useBulkApi) {
       return this;
     }
 
+    public Builder setBulkMaxRpcCount(int bulkMaxRpcCount) {
```

Comment:
Be consistent with the ordering. Use Count before Size everywhere or the other way around.